src/js/edu-benefits/1990-rjsf/config/form.js

```diff
@@ -24,6 +24,193 @@ const formConfig = {
   title: 'Apply for education benefits',
```

Comment:
I don't _think_ this is necessary, but I'll let somebody else make that decision. @J-Quag maybe? (If not, pull in whomever it should be.)

At any rate, this can be changed in another PR since it's not specifically related to the changes made here.